_docs-sources/intro/overview/use-cases.md

```diff
@@ -0,0 +1,8 @@
+# Use Cases
```

Comment:
"Use cases" sounds a little vague here. I'd prefer something that describes it in a more personalized way, such as "Gruntwork for Your Team" or "Ways to Use Gruntwork"